Fix:

aseprite/laf/os/osx/window.mm:408:47: error: no visible @interface for 'WindowOSXObjc' declares the selector 'convertPointFromScreen:'
--- aseprite/laf/os/osx/window.h.orig	2021-07-08 19:36:23.000000000 -0500
+++ aseprite/laf/os/osx/window.h	2021-07-16 00:30:51.000000000 -0500
@@ -52,6 +52,9 @@
 - (void)setMousePosition:(const gfx::Point&)position;
 - (BOOL)setNativeCursor:(os::NativeCursor)cursor;
 - (BOOL)canBecomeKeyWindow;
+#if MAC_OS_X_VERSION_MAX_ALLOWED < 101400
+- (NSPoint)convertPointFromScreen:(NSPoint)point NS_AVAILABLE_MAC(10_12);
+#endif
 @end
 
 using WindowOSXObjc_id = WindowOSXObjc*;
